docs/development.md

```diff
@@ -16,6 +16,34 @@ By default, this script will deploy a `postgresql` instance, the worker and the
 
 You can skip deployment and basic settings if you use the script.
 
+### MacOS Users
+
+If you are using MacOS to develop for OpenScanHub you will need to set up a different environment:
+
+1. You will need to install the latest Bash version using brew:
+    `brew install bash`
+2. Locate the new package:
+   `which bash` -> `/opt/homebrew/bin/bash`
```

Comment:
```suggestion
   `command -v bash` -> `/opt/homebrew/bin/bash`
```
I'd use a portable solution that does not require any extra programs like `which` instead.  I don't know if `which` installed by default on macOS but it definitely is not the case with some Linux distros.  Also, `command` is specified by POSIX: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html

Reply:
This is still not resolved because `command` is not the same as `command -v`. See the linked POSIX manual page in comment https://github.com/openscanhub/openscanhub/pull/29#discussion_r1255760844.